### PR TITLE
fix: solve 404 error by better resolving of path to studio

### PR DESCRIPTION
**Pull request by derberg**
Fixes https://github.com/asyncapi/cli/issues/244

Continuation of https://github.com/asyncapi/cli/pull/332 that initial contributor was not interested to continue

### CODE REVIEW

**Comment by derberg** [comment]
@Souvikns can you have a look?

**Review by Souvikns — APPROVED** [review]

File: src/models/Studio.ts [hunk 1]
```diff
@@ -45,8 +44,13 @@ export function start(filePath: string, port: number = DEFAULT_PORT): void {
   });
 
   const server = createServer((request, response) => {
+    //not all CLI users use npm. Some package managers put dependencies in different weird places
+    //this is why we need to first figure out where exactly is the index.html located 
+    //and then strip index.html from the path to point to directory with the rest of the studio
+    const indexLocation = require.resolve('@asyncapi/studio/build/index.html');
```
Comment (Souvikns) [inline src/models/Studio.ts:50]: This is looking for a global install studio right?

Comment (derberg) [reply src/models/Studio.ts:50]: for any closest
so if you use CLI as local dependency, this one will be picked up

**Pull request merged**